assets/cluster-version-operator/cluster-version-operator-deployment.yaml

```diff
@@ -83,13 +101,17 @@ spec:
             - mountPath: /etc/tls/serving-cert
               name: serving-cert
               readOnly: true
+            - mountPath: /var/payload
+              name: payload
           env:
             - name: NODE_NAME
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-            - name: EXCLUDE_MANIFESTS
-              value: internal-openshift-hosted
+            - name: CLUSTER_PROFILE
```

Comment:
So is `EXCLUDE_MANIFESTS` replaced by `CLUSTER_PROFILE`?

Reply:
That is correct. The profile defines what manifests are included/excluded